src/main/java/com/google/codeu/utils/CsvLoader.java

```diff
@@ -0,0 +1,32 @@
+package com.google.codeu.utils;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonArray;
+import com.google.gson.JsonObject;
+import java.util.Scanner;
+
+public class CsvLoader {
+
+  /**
+   * Loads data from a csv from the given scanner and returns a json array
+   */
+  public static JsonArray csvToJson(Scanner scanner) throws Exception {
+
+    JsonArray jsonArray = new JsonArray();
+    String headerLine = scanner.nextLine();
+    String[] headers = headerLine.split(",");
+    while(scanner.hasNextLine()) {
+      String line = scanner.nextLine();
+      String[] cells = line.split(",");
+
+      JsonObject obj = new JsonObject();
+      for (int i = 0; i < cells.length; i++) {
+        obj.addProperty(headers[i], cells[i]);
```

Comment:
nit: if you want to be extra careful, you could add a check that `i` is a valid index for `headers`

Reply:
What does this mean? Isn't i just like a temporary counter..? Why wouldn't it be a valid index for headers (whats headers)?

Reply:
The length of `headers` and `cells` could be different, depending on the CSV file (corrupted CSV file, which is possible if the file isn't generated but written by hand).

Reply:
I have added in an exception statement to check if all cell lengths are the same as header length